recipes/xtrabackuprb.rb

```diff
@@ -1,24 +1,30 @@
 include_recipe 'git'
 include_recipe 'percona::package_repo'
 
+if platform_family?('rhel') && node['platform_version'].to_i == 6
+  include_recipe 'yum-epel'
+  package 'libev'
```

Comment:
There's no need to install the ``libev`` package since that get's pulled in via the ``percona-xtrabackup`` package as a dependency. I recommend you just change this whole block to be:
``` ruby
include_recipe 'yum-epel' if node['platform_version'].to_i == 6
￼```